Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/MinimumFlowBypass/Controller.mo

```diff
@@ -0,0 +1,223 @@
+﻿within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.MinimumFlowBypass;
+block Controller
+  "Controller for chilled water minimum flow bypass valve"
+
+  parameter Integer nChi=3 "Total number of chillers";
+  parameter Modelica.SIunits.Time byPasSetTim = 300
+    "Time to reset minimum by-pass flow";
+  parameter Modelica.SIunits.VolumeFlowRate minFloSet[nChi] = {0.005, 0.005, 0.005}
+    "Minimum chilled water flow through each chiller";
+  parameter Modelica.SIunits.VolumeFlowRate maxFloSet[nChi]={0.025,0.025,0.025}
+    "Maximum chilled water flow through each chiller";
+  parameter Buildings.Controls.OBC.CDL.Types.SimpleController controllerType=
+    Buildings.Controls.OBC.CDL.Types.SimpleController.PID
+    "Type of controller"
+    annotation (Dialog(group="Controller"));
+  parameter Real k=1 "Gain of controller"
+    annotation (Dialog(group="Controller"));
+  parameter Modelica.SIunits.Time Ti=0.5 "Time constant of integrator block"
+    annotation (Dialog(group="Controller"));
+  parameter Modelica.SIunits.Time Td=0.1 "Time constant of derivative block"
```

Comment:
I would use 0 as default since HVAC controls seldom use derivative terms from my experience (hard to tune plus unnecessary considering the typical dynamics involved in HVAC control loops).